### PR TITLE
update instances of `/r` for `@Damage` for re-en.json

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -333,164 +333,164 @@
...
+                    "success": "<strong>Effect</strong> The bomb also deals  @Damage[1d4[persistent,piercing]] and @Damage[(1[splash])[poison]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round."
                 },
                 "Major": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*4d4)[persistent,piercing]]] and [[/r (4[splash])[poison]]]{4 poison splash damage}. The bomb's splash area is coated in vegetation, becoming difficult terrain for 1 minute.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 4d4[persistent,piercing]]] and [[/r (4[splash])[poison]]]{4 poison splash damage}. The bomb's splash area is coated in vegetation, becoming difficult terrain for 1 minute."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals2*@Damage[4d4[persistent,piercing]] and @Damage[(4[splash])[poison]]{4 poison splash damage}. The bomb's splash area is coated in vegetation, becoming difficult terrain for 1 minute.",
```
Comment (j-bs) [inline static/lang/re-en.json:398]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals 2*@Damage[4d4[persistent,piercing]] and @Damage[(4[splash])[poison]]{4 poison splash damage}. The bomb's splash area is coated in vegetation, becoming difficult terrain for 1 minute.",
```

File: static/lang/re-en.json [hunk 2]
```diff
@@ -333,164 +333,164 @@
...
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(3[splash])[bludgeoning]]{3 bludgeoning splash damage}, Medium or smaller targets are pushed 5 feet away from you."
                 },
                 "Lesser": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (1[splash])[bludgeoning]]]{1 bludgeoning splash damage}, Tiny or smaller targets are pushed 5 feet away from you, and the target is knocked @UUID[Compendium.pf2e.conditionitems.Item.j91X7x0XSomq8d60]{Prone}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r (1[splash])[bludgeoning]]]{1 bludgeoning splash damage}, Tiny or smaller targets are pushed 5 feet away from you."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(1[splash])[bludgeoning]]1 bludgeoning splash damage}, Tiny or smaller targets are pushed 5 feet away from you, and the target is knocked @UUID[Compendium.pf2e.conditionitems.Item.j91X7x0XSomq8d60]{Prone}.",
```
Comment (j-bs) [inline static/lang/re-en.json:429]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(1[splash])[bludgeoning]]{1 bludgeoning splash damage}, Tiny or smaller targets are pushed 5 feet away from you, and the target is knocked @UUID[Compendium.pf2e.conditionitems.Item.j91X7x0XSomq8d60]{Prone}.",
```

File: static/lang/re-en.json [hunk 3]
```diff
@@ -333,164 +333,164 @@
...
                 }
             },
             "RedpitchBomb": {
                 "Greater": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*3d4)[persistent,fire]]] and [[/r (3[splash])[fire]]]{3 fire splash damage}. On a critical hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.i3OJZU2nk64Df3xm]{Clumsy 2} until the start of your next turn.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3d4[persistent,fire]]] and [[/r (3[splash])[fire]]]{3 fire splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals 2*@Damage[3d4[persistent,fire]] and @Damage[(3[splash])[fire]]{3 fire splash damage}. On a critical hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.i3OJZU2nk64Df3xm]{Clumsy 2} until the start of your next turn.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[3d4[persistent,fire]] and @Damage[(3[splash])[fire]]3 fire splash damage}."
```
Comment (j-bs) [inline static/lang/re-en.json:444]:
```suggestion
                    "success": "<strong>Effect</strong> The bomb also deals @Damage[3d4[persistent,fire]] and @Damage[(3[splash])[fire]]{3 fire splash damage}."
```

File: static/lang/re-en.json [hunk 4]
```diff
@@ -513,66 +513,66 @@
...
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(4[splash])[mental]]{4 mental splash damage} and the target must succeed at a @Check[type:flat|dc:5] before taking actions with the concentrate trait, this lasts until the end of its next turn."
                 },
                 "Moderate": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2[splash])[mental]]]{2 mental splash damage} and the target must succeed at a @Check[type:flat|dc:5] before taking actions with the concentrate trait, this lasts for 1 minute.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r (2[splash])[mental]]]{2 mental splash damage} and the target must succeed at a @Check[type:flat|dc:5] before taking actions with the concentrate trait, this lasts until the end of its next turn."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2[splash])[mental]][mental]]]{2 mental splash damage} and the target must succeed at a @Check[type:flat|dc:5] before taking actions with the concentrate trait, this lasts for 1 minute.",
```
Comment (j-bs) [inline static/lang/re-en.json:560]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2[splash])[mental]]{2 mental splash damage} and the target must succeed at a @Check[type:flat|dc:5] before taking actions with the concentrate trait, this lasts for 1 minute.",
```

File: static/lang/re-en.json [hunk 5]
```diff
@@ -3333,9 +3333,9 @@
...
-                        "FourteenthLevel": "The creature takes [[/r 3d6[persistent,void]]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}.",
-                        "TwentiethLevel": "The creature takes [[/r 4d6[persistent,void]]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}."
+                        "Base": "The creature takes @Damage[2d6[persistent,void]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}.",
+                        "FourteenthLevel": "The creature takes  @Damage[3d6[persistent,void]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}.",
+                        "TwentiethLevel": "The creature takes  @Damage[4d6[persistent,void]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}."
```
Comment (j-bs) [inline static/lang/re-en.json:3338]:
```suggestion
                        "FourteenthLevel": "The creature takes @Damage[3d6[persistent,void]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}.",
                        "TwentiethLevel": "The creature takes @Damage[4d6[persistent,void]] damage. While the creature has this persistent damage, it's also @UUID[Compendium.pf2e.conditionitems.Item.e1XGnhKNSQIm5IXg]{Stupefied 1}."
```

File: static/lang/re-en.json [hunk 6]
```diff
@@ -333,164 +333,164 @@
...
                 },
                 "Lesser": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*1d4)[persistent,piercing]]] and [[/r (1[splash])[poison]]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 1d4[persistent,piercing]]] and [[/r (1[splash])[poison]]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals  2*@Damage[1d4[persistent,piercing]] and @Damage[(1[splash])[poison]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round.",
+                    "success": "<strong>Effect</strong> The bomb also deals  @Damage[1d4[persistent,piercing]] and @Damage[(1[splash])[poison]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round."
```
Comment (j-bs) [inline static/lang/re-en.json:395]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals 2*@Damage[1d4[persistent,piercing]] and @Damage[(1[splash])[poison]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round.",
                    "success": "<strong>Effect</strong> The bomb also deals @Damage[1d4[persistent,piercing]] and @Damage[(1[splash])[poison]]{1 poison splash damage}. One square of the target creature's space is coated in vegetation, becoming difficult terrain for 1 round."
```

File: static/lang/re-en.json [hunk 7]
```diff
@@ -333,164 +333,164 @@
...
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[2[bleed]] and @Damage[(2[splash])[slashing]]{2 slashing splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[2[bleed]] and @Damage[(2[splash])[slashing]]{2 slashing splash damage}."
                 }
             },
             "MudBomb": {
                 "Greater": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (3[splash])[bludgeoning]]]{3 bludgeoning splash damage}. The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} until the start of your next turn.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r (3[splash])[bludgeoning]]]{3 bludgeoning splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(3[splash])[bludgeoning]]. The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} until the start of your next turn.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(3[splash])[bludgeoning]]."
```
Comment (j-bs) [inline static/lang/re-en.json:355]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(3[splash])[bludgeoning]]{3 bludgeoning splash damage}. The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} until the start of your next turn.",
                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(3[splash])[bludgeoning]]{3 bludgeoning splash damage}."
```

**Review by j-bs — COMMENTED** [review]
Found a few errors 👀

**Comment by Cuingamehtar** [comment]
A lot of the changes break the formulas here. Where there is doubling in the formula, it's outright removed or has to be done manually now (all of persistent damage strings and bombs).
Nagaji Venom Spit loses its persistent damage.

It's persist**e**nt, not persist**a**nt in damage labels. And if you are writing custom label for persistent damage link, it will automatically fill "n persistent element".

Also, a lot of those are not used in the current version. I imagine those are still there for legacy reasons, or for translations, which won't be affected by these changes.

**Comment by ammalagonc** [comment]
Thanks for catching those typos, @j-bs. Please check again when you get a moment :)

**Comment by ammalagonc** [comment]
@Cuingamehtar - I fixed the spelling on persistent. 

I will check later to see what the best approach is for doubling damage using `@Damage`. I was going off an existing example, but apparently, that doesn't work.

**Comment by Cuingamehtar** [comment]
@ammalagonc Use brackets. @Damage[(2*2d4)[persistent,fire]] works.

**Comment by ammalagonc** [comment]
Made corrections to the instances of critical damage (2*) in the file.

Please check when you get a moment.

Thanks again @Cuingamehtar and @j-bs for your reviews and making sure this is up to par :)

File: static/lang/re-en.json [hunk 8]
```diff
@@ -193,134 +193,134 @@
         "BombNotes": {
             "AcidFlask": {
                 "Greater": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*3d6)[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3d6[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[3d6[persistent,acid]]and @Damage[(3[splash])[acid]]{3 acid splash damage}.",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:196]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[3d6[persistent,acid]] and @Damage[(3[splash])[acid]]{3 acid splash damage}.",
```

File: static/lang/re-en.json [hunk 9]
```diff
@@ -1193,252 +1193,252 @@
...
             "Acid2d6": {
-                "criticalSuccess": "[[/r (2*2d6)[persistent,acid]]]",
-                "success": "[[/r 2d6[persistent,acid]]]"
+                "criticalSuccess": "@Damage[(2*2d6)[persistent,acid]]",
+                "success": "*@Damage[2d6[persistent,acid]]"
             },
             "Acid2d6+2": {
-                "criticalSuccess": "[[/r (2*2d6+4)[persistent,acid]]]",
-                "success": "[[/r (2d6+2)[persistent,acid]]]"
+                "criticalSuccess": "@Damage[(2*(2d6+4))[persistent,acid]]",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1208]:
```suggestion
                "criticalSuccess": "@Damage[(2*2d6+4)[persistent,acid]]",
```
This +4 is already doubled.

File: static/lang/re-en.json [hunk 10]
```diff
@@ -1193,252 +1193,252 @@
...
             "Bleed2": {
-                "criticalSuccess": "[[/r 4[bleed]]]",
-                "success": "[[/r 2[bleed]]]"
+                "criticalSuccess": "@Damage[4[bleed]]",
+                "success": "@Damage[2[bleed]]"
             },
             "Bleed2d10": {
-                "criticalSuccess": "[[/r (2*2d10)[bleed]]]",
-                "success": "[[/r 2d10[bleed]]]"
+                "criticalSuccess": "@Damage[(2*1d10)[bleed]]",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1240]:
```suggestion
                "criticalSuccess": "@Damage[(2*2d10)[bleed]]",
```

File: static/lang/re-en.json [hunk 11]
```diff
@@ -1193,252 +1193,252 @@
...
+                "success": "@Damage[1d10[bleed]]"
             },
             "Bleed2d12": {
-                "criticalSuccess": "[[/r (2*2d12)[bleed]]]",
-                "success": "[[/r 2d12[bleed]]]"
+                "criticalSuccess": "@Damage[(2*1d12)[bleed]]",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1244]:
```suggestion
                "criticalSuccess": "@Damage[(2*2d12)[bleed]]",
```

File: static/lang/re-en.json [hunk 12]
```diff
@@ -1193,252 +1193,252 @@
...
+                "success": "@Damage[1d12[bleed]]"
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1245]:
```suggestion
                "success": "@Damage[2d12[bleed]]"
```

File: static/lang/re-en.json [hunk 13]
```diff
@@ -1193,252 +1193,252 @@
...
-                "criticalSuccess": "[[/r 4[bleed]]]",
-                "success": "[[/r 2[bleed]]]"
+                "criticalSuccess": "@Damage[4[bleed]]",
+                "success": "@Damage[2[bleed]]"
             },
             "Bleed2d10": {
-                "criticalSuccess": "[[/r (2*2d10)[bleed]]]",
-                "success": "[[/r 2d10[bleed]]]"
+                "criticalSuccess": "@Damage[(2*1d10)[bleed]]",
+                "success": "@Damage[1d10[bleed]]"
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1241]:
```suggestion
                "success": "@Damage[2d10[bleed]]"
```

File: static/lang/re-en.json [hunk 14]
```diff
@@ -1193,252 +1193,252 @@
...
             "Fire4d10": {
-                "criticalSuccess": "[[/r (2*4d10)[persistent,fire]]]",
-                "success": "[[/r 4d10[persistent,fire]]]"
+                "criticalSuccess": "@Damage[(2*4d10)[persistent,fire]]",
+                "success": "@Damage[4d10[persistent,fire]]"
             },
             "Force1d12": {
-                "criticalSuccess": "[[/r (2*1d12)[persistent,force]]]",
-                "success": "[[/r 1d12[persistent,force]]]"
+                "criticalSuccess": "@Damage[(2*1d12)[persistent,fire]]",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1356]:
```suggestion
                "criticalSuccess": "@Damage[(2*1d12)[persistent,force]]",
```

File: static/lang/re-en.json [hunk 15]
```diff
@@ -1193,252 +1193,252 @@
...
+                "success": "@Damage[1d12[persistent,fire]]"
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1357]:
```suggestion
                "success": "@Damage[1d12[persistent,force]]"
```

File: static/lang/re-en.json [hunk 16]
```diff
@@ -1193,252 +1193,252 @@
...
             },
             "Force2d12": {
-                "criticalSuccess": "[[/r (2*2d12)[persistent,force]]]",
-                "success": "[[/r 2d12[persistent,force]]]"
+                "criticalSuccess": "@Damage[(2*2d12)[persistent,fire]]",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1360]:
```suggestion
                "criticalSuccess": "@Damage[(2*2d12)[persistent,force]]",
```

File: static/lang/re-en.json [hunk 17]
```diff
@@ -1193,252 +1193,252 @@
...
+                "success": "@Damage[2d12[persistent,fire]]"
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1361]:
```suggestion
                "success": "@Damage[2d12[persistent,force]]"
```

File: static/lang/re-en.json [hunk 18]
```diff
@@ -1193,252 +1193,252 @@
...
             "Negative2d10": {
-                "criticalSuccess": "[[/r (2*2d10)[persistent,void]]]",
-                "success": "[[/r 2d10[persistent,void]]]"
+                "criticalSuccess": "@Damage[(2*2d10)[persistent,void]]",
+                "success": "*@Damage[2d10[persistent,void]]"
             },
             "Piercing2d8+8": {
-                "criticalSuccess": "[[/r (2*2d8+16)[persistent,piercing]]]",
-                "success": "[[/r (2d8+8)[persistent,piercing]]]"
+                "criticalSuccess": "@Damage[(2*(2d8+16))[persistent,piercing]]",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:1400]:
```suggestion
                "criticalSuccess": "@Damage[(2*2d8+16)[persistent,piercing]]",
```
+16 is already doubled

File: static/lang/re-en.json [hunk 19]
```diff
@@ -3348,7 +3348,7 @@
             },
             "Nagaji": {
                 "VenomSpit": {
-                    "CriticalNote": "The target takes [[/r (1+@weapon.system.runes.striking)[persistent,poison]]]{persistent poison damage} equal to the number of weapon damage dice",
+                    "CriticalNote": "The target takes @Damage[(1+@weapon.system.runes.striking)[poison]]{persistent poison damage} equal to the number of weapon damage dice",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:3351]:
```suggestion
                    "CriticalNote": "The target takes @Damage[(1+@weapon.system.runes.striking)[persistent,poison]]{persistent poison damage} equal to the number of weapon damage dice",
```

**Review by Cuingamehtar — CHANGES_REQUESTED** [review]
Please be more attentive to the formulas and damage types.

**Comment by ammalagonc** [comment]
@Cuingamehtar - Changes made.

Please check when you get a moment. Thanks again for all you do.

File: static/lang/re-en.json [hunk 20]
```diff
@@ -193,134 +193,134 @@
...
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(4[splash])[piercing]]{4 piercing splash damage} and the splash creates crystals that are @UUID[Compendium.pf2e.equipment-srd.Item.7fSnvJ2xoSfa6JXD]{Caltrops} on level surfaces and handholds on vertical surfaces, granting +1 item bonus to the next Athletics check to Climb."
                 }
             },
             "DreadAmpoule": {
                 "Greater": {
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r (3[splash])[mental]]]{3 mental splash damage} and the target is @UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 1} (@UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 2} on a critical)."
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(5[splash])[mental]]{3 mental splash damage} and the target is @UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 1} (@UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 2} on a critical)."
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:278]:
```suggestion
                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(3[splash])[mental]]{3 mental splash damage} and the target is @UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 1} (@UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 2} on a critical)."
```

File: static/lang/re-en.json [hunk 21]
```diff
@@ -193,134 +193,134 @@
         "BombNotes": {
             "AcidFlask": {
                 "Greater": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*3d6)[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3d6[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[3d6[persistent,acid]] and @Damage[(3[splash])[acid]]{3 acid splash damage}.",
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:196]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*3d6)[persistent,acid]] and @Damage[(3[splash])[acid]]{3 acid splash damage}.",
```

File: static/lang/re-en.json [hunk 22]
```diff
@@ -3132,7 +3132,7 @@
                     "Note": "On a critical hit, a seedpod bursts, issuing forth a tangle of vegetation that imposes a -10-foot circumstance penalty on the target's Speed for 1 round. @UUID[Compendium.pf2e.feat-effects.Item.wQDHpOKY3GZqvS2v]{Effect: Seedpod}"
                 },
                 "ThornedSeedpod": {
-                    "Note": "When you critically succeed at a seedpod Strike, vines twist and stab your foe, dealing [[/r 1d4[persistent,piercing]]]"
+                    "Note": "When you critically succeed at a seedpod Strike, vines twist and stab your foe, dealing @Damage[1d4[persistent,piercing]]{1d4 persistent piercing}"
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:3135]:
```suggestion
                    "Note": "When you critically succeed at a seedpod Strike, vines twist and stab your foe, dealing @Damage[1d4[persistent,piercing]]{1d4 persistent piercing damage}."
```

File: static/lang/re-en.json [hunk 23]
```diff
@@ -2796,7 +2796,7 @@
                     "Note": "If you roll a failure or critical failure on a saving throw against a harmful effect, you get a success instead."
                 },
                 "TorchGoblin": {
-                    "Note": "As long as you are suffering persistent fire damage, all your melee attacks against adjacent creatures deal an additional [[/r 1[fire]]]{1 Fire Damage} per weapon damage die."
+                    "Note": "As long as you are suffering persistent fire damage, all your melee attacks against adjacent creatures deal an additional @Damage[1[fire]]{1 Fire Damage} per weapon damage die."
```
Comment (Cuingamehtar) [inline static/lang/re-en.json:2799]:
```suggestion
                    "Note": "As long as you are suffering persistent fire damage, all your melee attacks against adjacent creatures deal an additional @Damage[1[fire]]{1 fire damage} per weapon damage die."
```

**Review by Cuingamehtar — CHANGES_REQUESTED** [review]
A couple of mistakes I've missed on previous review.

The rest seems fine.

File: static/lang/re-en.json [hunk 24]
```diff
@@ -333,164 +333,164 @@
...
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*4)[bleed]]] and [[/r (4[splash])[slashing]]]{4 slashing splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 4[bleed]]] and [[/r (4[splash])[slashing]]]{4 slashing splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*4)[bleed]] and @Damage[(4[splash])[slashing]]{4 slashing splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[4[bleed]] and @Damage[(4[splash])[slashing]]{4 slashing splash damage}."
                 },
                 "Moderate": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*2)[bleed]]] and [[/r (2[splash])[slashing]]]{2 slashing splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 2[bleed]]] and [[/r (2[splash])[slashing]]]{2 slashing splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[2[bleed]] and @Damage[(2[splash])[slashing]]{2 slashing splash damage}.",
```
Comment (j-bs) [inline static/lang/re-en.json:348]:
```suggestion
                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*2)[bleed]] and @Damage[(2[splash])[slashing]]{2 slashing splash damage}.",
```
Spotted another cheeky typo on a crit 2x entry.

Looks like the last one.

File: static/lang/re-en.json [hunk 25]
```diff
@@ -193,134 +193,134 @@
         "BombNotes": {
             "AcidFlask": {
                 "Greater": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*3d6)[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3d6[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*3d6)[persistent,acid]] and @Damage[(3[splash])[acid]]{3 acid splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[(3d6[persistent,acid]]{3d6 persistent acid} and @Damage[(3[splash])[acid]]{3 acid splash damage}."
```
Comment (TikaelSol) [inline static/lang/re-en.json:197]:
```suggestion
                    "success": "<strong>Effect</strong> The bomb also deals @Damage[3d6[persistent,acid]] and @Damage[(3[splash])[acid]]{3 acid splash damage}."
```

File: static/lang/re-en.json [hunk 26]
```diff
@@ -193,134 +193,134 @@
...
                 },
                 "Lesser": {
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*1d6)[persistent,acid]]] and [[/r (1[splash])[acid]]]{1 acid splash damage}.",
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 1d6[persistent,acid]]] and [[/r (1[splash])[acid]]]{1 acid splash damage}."
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals @Damage[(2*1d6)[persistent,acid]] and @Damage[(1[splash])[acid]]{1 acid splash damage}.",
+                    "success": "<strong>Effect</strong> The bomb also deals @Damage[1d6[persistent,acid]]{1d6 persistent acid} and @Damage[(1[splash])[acid]]{1 acid splash damage}."
```
Comment (TikaelSol) [inline static/lang/re-en.json:201]:
```suggestion
                    "success": "<strong>Effect</strong> The bomb also deals @Damage[1d6[persistent,acid]] and @Damage[(1[splash])[acid]]{1 acid splash damage}."
```

File: static/lang/re-en.json [hunk 27]
```diff
@@ -158,7 +158,7 @@
                 "SunNote": "The target is @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} for 1 round."
             },
             "DaemonForm": {
-                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional [[/r {1d6}]]{1d6 damage}, and you take the same amount of damage"
+                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]]{1d6 damage}, and you take the same amount of damage"
```
Comment (TikaelSol) [inline static/lang/re-en.json:161]:
```suggestion
                "CeustodaemonDamageNote": "Any successful jaws or claw Strike deals an additional @Damage[1d6[untyped]] damage, and you take the same amount of damage"
```

File: static/lang/re-en.json [hunk 28]
```diff
@@ -985,7 +985,7 @@
             "PoltergeistTelekineticStorm": {
                 "MultiAttackLabel": "Telekinetic Storm - Multiple Targets",
                 "MultipleTargets": "Multiple Targets",
-                "SingleAttackEffect": "The attack deals [[/r {1d12}]]{1d12 damage} on a failure.",
+                "SingleAttackEffect": "The attack deals @Damage[1d12[untyped]]{1d12 damage} on a failure.",
```
Comment (TikaelSol) [inline static/lang/re-en.json:988]:
```suggestion
                "SingleAttackEffect": "The attack deals @Damage[1d12[untyped]] damage on a failure.",
```

File: static/lang/re-en.json [hunk 29]
```diff
@@ -2902,7 +2902,7 @@
             },
             "Hobgoblin": {
                 "AgonizingRebuke": {
-                    "Note": "When you successfully Demoralize a foe, that foe takes [[/r 1d4[mental]]]{1d4 mental damage} at the start of each of its turns as long as it remains Frightened and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to [[/r 2d4[mental]]]{2d4}, and if you have legendary proficiency, the damage increases to [[/r 3d4[mental]]]{3d4}."
+                    "Note": "When you successfully Demoralize a foe, that foe takes @Damage[1d4[mental]]{1d4 mental damage} at the start of each of its turns as long as it remains Frightened and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to @Damage[2d4[mental]]{2d4}, and if you have legendary proficiency, the damage increases to @Damage[3d4[mental]]{3d4}."
```
Comment (TikaelSol) [inline static/lang/re-en.json:2905]:
```suggestion
                    "Note": "When you successfully Demoralize a foe, that foe takes @Damage[(ternary(gte(@actor.system.skills.intimidation.rank,4),3,ternary(gte(@actor.system.skills.intimidation.rank,3),2,1))d4)[mental]]{1d4 mental damage} at the start of each of its turns as long as it remains Frightened and continues to engage in combat with you. If you have master proficiency in Intimidation, the damage increases to 2d4, and if you have legendary proficiency, the damage increases to 3d4."
```

File: static/lang/re-en.json [hunk 30]
```diff
@@ -3132,7 +3132,7 @@
                     "Note": "On a critical hit, a seedpod bursts, issuing forth a tangle of vegetation that imposes a -10-foot circumstance penalty on the target's Speed for 1 round. @UUID[Compendium.pf2e.feat-effects.Item.wQDHpOKY3GZqvS2v]{Effect: Seedpod}"
                 },
                 "ThornedSeedpod": {
-                    "Note": "When you critically succeed at a seedpod Strike, vines twist and stab your foe, dealing [[/r 1d4[persistent,piercing]]]"
+                    "Note": "When you critically succeed at a seedpod Strike, vines twist and stab your foe, dealing @Damage[1d4[persistent,piercing]]{1d4 persistent piercing damage}."
```
Comment (TikaelSol) [inline static/lang/re-en.json:3135]:
```suggestion
                    "Note": "When you critically succeed at a seedpod Strike, vines twist and stab your foe, dealing @Damage[1d4[persistent,piercing]] damage."
```

File: static/lang/re-en.json [hunk 31]
```diff
@@ -3348,7 +3348,7 @@
             },
             "Nagaji": {
                 "VenomSpit": {
-                    "CriticalNote": "The target takes [[/r (1+@weapon.system.runes.striking)[persistent,poison]]]{persistent poison damage} equal to the number of weapon damage dice",
+                    "CriticalNote": "The target takes @Damage[(1+@weapon.system.runes.striking)[persistent,poison]]{persistent poison damage} equal to the number of weapon damage dice",
```
Comment (TikaelSol) [inline static/lang/re-en.json:3351]:
```suggestion
                    "CriticalNote": "The target takes @Damage[(@weapon.system.damage.dice)[persistent,poison]]{persistent poison damage} equal to the number of weapon damage dice",
```

**Pull request merged**